perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AuthzResolverBlImpl.java

```diff
@@ -1015,6 +1015,18 @@ public static boolean isPerunAdmin(PerunSession sess) {
 		return sess.getPerunPrincipal().getRoles().getRolesNames();
 	}
 
+	/**
+	 * Get all User's roles. Role is defined as a name, translation table is in Role class.
+	 *
+	 * @param sess perun session
+	 * @param user User
+	 * @return list of integers, which represents role from enum Role.
+	 */
+	public static List<String> getUserRoles(PerunSession sess,User user) throws InternalErrorException {
```

Comment:
There is a mismatch in method name. In above layer ``getUserRoleNames()`` is used, but here it's ``getUserRoles()``. Please use same name everywhere.